put_get_test.go

```diff
@@ -273,6 +276,47 @@ func TestPutWithAutoCompressFalse(t *testing.T) {
 		if strings.Contains(file, "data.txt.gz") {
 			t.Fatalf("should not contain file. got: %v", file)
 		}
+
+		// GET test
+		streamBuf := new(bytes.Buffer)
+		ctx := WithFileTransferOptions(context.Background(), &SnowflakeFileTransferOptions{getFileToStream: true})
+		ctx = WithFileGetStream(ctx, &streamBuf)
+		sql := fmt.Sprintf("get @~/test_put_uncompress_file/data.txt 'file://%v'", tmpDir)
+		sqlText = strings.ReplaceAll(sql, "\\", "\\\\")
+		rows2 := dbt.mustQueryContext(ctx, sqlText)
+		defer rows2.Close()
+		for rows2.Next() {
+			if err = rows2.Scan(&file, &s1, &s2, &s3); err != nil {
+				t.Error(err)
```

Comment:
Can you use `assertNilF(t, err)` and similar asssertXXX? And fix in this test function?

Reply:
Updated test.